src/App.js

```diff
@@ -10,10 +11,37 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    tabs,
+    selectedTab: tabs[0],
+    tabContent: tabs[0].content,
+    active: tabs[0].id,
+  }
```

Comment:
```suggestion
  state = {
    selectedTab: tabs[0],
  }
```
все остальное лишнее